### PR TITLE
Hm 03

### PR DESCRIPTION
**Pull request by a1442456**

### CODE REVIEW

File: HomeWorkSolution/Task2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            int result = int.Parse(Console.ReadLine());          
+            if (result == WorkWithTwoNumbers(num1, num2, sign))
+                Console.WriteLine("Done! You're goddamn right. Totally sure you can type binary code instead of c#. Think about it! :)");
+            else if(result > WorkWithTwoNumbers(num1, num2, sign))
+                Console.WriteLine("Less than you think! :(");
+            else
+                Console.WriteLine("More than you think! :(");
+            Console.ReadLine();
+        }
+        /// <summary>
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: add one empty line under

File: HomeWorkSolution/Task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        static void Main(string[] args)
+        {   
+            int num1 = InputNumber();
+            int num2 = InputNumber();
+            Console.WriteLine("Enter a math sign");
+            char sign = Console.ReadLine()[0]; //Read 1st symbol of an entered string
+
+            Console.WriteLine("Try to calculate it yourself!");
+            Console.WriteLine("Your answer is:");
+            int result = int.Parse(Console.ReadLine());
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: type.**Parse()** - great solution!
I would suggest to make a small investigation - take a look this method:  type.**TyParse()**
Decompile both methods and tell me which one needs to use.

File: HomeWorkSolution/Task2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+            else if(result > WorkWithTwoNumbers(num1, num2, sign))
+                Console.WriteLine("Less than you think! :(");
+            else
+                Console.WriteLine("More than you think! :(");
+            Console.ReadLine();
+        }
+        /// <summary>
+        /// Return a number entered in Console
+        /// </summary>
+        /// <returns></returns>
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: returns section also should be documented :). anyway - great job!

File: HomeWorkSolution/Task2/Program.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        /// <param name="sign">Math Sign</param>
+        /// <returns></returns>
+        static int WorkWithTwoNumbers(int num1, int num2, char sign)
+        {
+            if (sign == '-')
+                return num1 - num2;
+            else if (sign == '+')
+                return num1 + num2;
+            else
+                throw new Exception("This sign can't be enterpritate as a math sign :(");
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: great!

File: HomeWorkSolution/Task2/Program.cs [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        /// <returns></returns>
+        static int InputNumber()
+        {
+            Console.WriteLine(string.Concat("Enter number:"));
+            int number = int.Parse(Console.ReadLine());
+            return number;
+        }
+
+        /// <summary>
+        /// Returns result of operation between two numbers with a math sign
```
Comment (VadzimPapko) [inline HomeWorkSolution/Task2/Program.cs]: at the end - the Dot character is required

**Comment by VadzimPapko** [comment]
@a1442456  ups,  some merge conflicts. Please fix and close PR yourself

File: HomeWorkSolution/Task2/Program.cs (hunk 1 above)
Comment (a1442456) [reply HomeWorkSolution/Task2/Program.cs]: Fixed

File: HomeWorkSolution/Task2/Program.cs (hunk 3 above)
Comment (a1442456) [reply HomeWorkSolution/Task2/Program.cs]: Fixed

File: HomeWorkSolution/Task2/Program.cs (hunk 4 above)
Comment (a1442456) [reply HomeWorkSolution/Task2/Program.cs]: Thanks :)

File: HomeWorkSolution/Task2/Program.cs (hunk 5 above)
Comment (a1442456) [reply HomeWorkSolution/Task2/Program.cs]: Fixed

File: HomeWorkSolution/Task2/Program.cs (hunk 2 above)
Comment (a1442456) [reply HomeWorkSolution/Task2/Program.cs]: Ok i'll do my best :)

**Pull request closed without merging**